### PR TITLE
Adding users and messages

### PR DESCRIPTION
**Pull request by mengyushi**
Users:
- new
- create
- show
(TODO: EDIT)

Message:
- new
- create
- show
(TODO: EDIT)

### CODE REVIEW

**Comment by FEIHONGQIAN** [comment]
reviewed

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+	def create
+	  @message = Message.new(message_params)
+	  @message.open = true
+	  @message.save
+	  redirect_to @message
+	end
+
+	def show
+      @message = Message.find(params[:id])
```
Comment (hanroro) [inline app/controllers/messages_controller.rb:14]: Miss a tab?

**Review by FEIHONGQIAN — APPROVED** [review]
reviewed

**Review by hanroro — APPROVED** [review]
Nice.

**Pull request merged**